src/app/result.ts

```diff
@@ -1,8 +1,11 @@
 import { CeleryBackend } from "../backends";
 
+type AsyncResultStatus = "PENDING" | "SUCCESS" | "FAILURE" | "TIMEOUT";
```

Comment:
Can you remove "TIMEOUT" in status? I guess there's no need to represent timeout as status.